knotx-core/knotx-template-engine/src/main/java/com/cognifide/knotx/engine/TemplateEngineVerticle.java

```diff
@@ -41,22 +43,26 @@
 
     private String serviceName;
 
+    private HttpClient httpClient;
+
     @Override
     public void init(Vertx vertx, Context context) {
         super.init(vertx, context);
         JsonObject config = config().getJsonObject("config");
         this.serviceName = config.getString("service.name");
 
         configuration = new TemplateEngineConfiguration(config);
-        templateEngine = new TemplateEngine(this.vertx, configuration);
+        final HttpClientOptions httpClientOptions = new HttpClientOptions().setMaxPoolSize(1000).setKeepAlive(false);
```

Comment:
I think we should have this configurable on JSON level. Like for repository Verticle.